src/pages/calculator.jsx

```diff
@@ -0,0 +1,18 @@
+import Calculator from '../components/Calculator';
+
+export default function CalcPage() {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        padding: '20px',
+      }}
+    >
+      <div>
+        <h4>Let&apos;s do some math!</h4>
+      </div>
+      <Calculator />
```

Comment:
- [ ] The only issue your calculator have is to divide a number by `zero` let us say `5/0` normally it has to display `error` in the screen but currently am getting this
![e](https://user-images.githubusercontent.com/44745855/129612546-1d3d31d0-35f4-4959-bd4b-fae6cd12523a.png)

Please fix that you can check what am talking about on your normal calculator!

Reply:
I have fixed this error now.

Thank you for the review